synapse/lib/stormlib/aha.py

```diff
@@ -0,0 +1,164 @@
+import synapse.common as s_common
+import synapse.lib.stormtypes as s_stormtypes
+
+@s_stormtypes.registry.registerLib
+class AhaPoolLib(s_stormtypes.Lib):
+    '''
+    A Storm Library for interacting with AHA service pools.
+    '''
+    _storm_lib_path = ('aha', 'pool')
+
+    def getObjLocals(self):
+        return {
+            'add': self.add,
+            'del': self._del,
+            'get': self.get,
+            'list': self.list,
+        }
+
+    async def add(self, name):
+        self.runt.reqAdmin()
+        proxy = await self.runt.snap.core.reqAhaProxy()
+        poolinfo = {'creator': self.runt.user.iden}
+        poolinfo = await proxy.addAhaPool(name, poolinfo)
+        return AhaPool(self.runt, poolinfo)
+
+    async def _del(self, name):
+        self.runt.reqAdmin()
+        proxy = await self.runt.snap.core.reqAhaProxy()
+        return await proxy.delAhaPool(name)
+
+    async def get(self, name):
+        self.runt.reqAdmin()
+        proxy = await self.runt.snap.core.reqAhaProxy()
+        poolinfo = await proxy.getAhaPool(name)
+        if poolinfo is not None:
```

Comment:
Should we raise a `NoSuchName` exception if the pool doesn't exist?

Reply:
I actually prefer this over raising an exception like we do in some `get` methods.